nemo/collections/nlp/parts/megatron_trainer_builder.py

```diff
@@ -83,7 +95,13 @@ def _plugins(self) -> list:
     def create_trainer(self) -> Trainer:
         strategy = self._training_strategy()
         plugins = self._plugins()
-        return Trainer(plugins=plugins, strategy=strategy, **self.cfg.trainer)
+        trainer = Trainer(plugins=plugins, strategy=strategy, **self.cfg.trainer)
+
+        if self.cfg.model.resume_from_checkpoint is not None:
+            trainer.ckpt_path = self.cfg.model.resume_from_checkpoint
+            logging.info(f'Resuming training from checkpoint: {trainer.ckpt_path}')
```

Comment:
I think I made a PR recently to handle this `trainer.ckpt_path` in `exp_manager.py` itself. I guess it works here too, but take a look at #7335. Let me know if you are having issues using `resume_from_checkpoint` on main branch.

Reply:
thanks, removed this